src/openapi/streaming/subscription.js

```diff
@@ -349,7 +349,11 @@ function onUnsubscribeSuccess(referenceId, response) {
  */
 function onUnsubscribeError(referenceId, response) {
     if (referenceId !== this.referenceId) {
-        log.error(LOG_AREA, 'Received an error response for unsubscribing a subscription that has afterwards been reset - ignoring');
+        log.error(

```

Comment:
I'm not sure this should be an error.. 

also the message is incorrect - a unsubscribe,  subscribe could cause a change in reference id - the same as a reset.
when a reset occurs do we unsubscribe to be sure? because the subscription might have been cleaned up already

Maybe change it to a warning? but adding more info can't hurt.

Reply:
My initial intention was to have deeper view inside the log alerts we get for this scenarios and check why exactly they are happening. However, I might as well check it by manually reviewing kibana logs on WARN level instead of waiting for log alerts.